README.md

```diff
@@ -226,6 +226,29 @@ In summary (from https://stackoverflow.com/a/69673312):
 alembic revision --autogenerate -m "<msg>"
 ```
 
+### Pre-commit Hooks
+
+To ensure code quality and consistency, OpenAdapt uses pre-commit hooks. These hooks will be executed automatically before each commit to perform various checks and validations on your codebase.
+
+The following pre-commit hooks are used in OpenAdapt:
+
+- [check-yaml](https://github.com/pre-commit/pre-commit-hooks#check-yaml): Validates the syntax and structure of YAML files.
+- [end-of-file-fixer](https://github.com/pre-commit/pre-commit-hooks#end-of-file-fixer): Ensures that files end with a newline character.
+- [trailing-whitespace](https://github.com/pre-commit/pre-commit-hooks#trailing-whitespace): Detects and removes trailing whitespace at the end of lines.
+- [isort](https://github.com/PyCQA/isort): Sorts Python import statements in a consistent and standardized manner.
+
+To set up the pre-commit hooks, follow these steps:
+
+1. Navigate to the root directory of your OpenAdapt repository.
+
+3. Run the following command to install the hooks:
```

Comment:
Is there a step 2 that is missing, or is this a typo?

Reply:
typo 👍

Reply:
resolved in https://github.com/OpenAdaptAI/OpenAdapt/pull/319/commits/7099d7f9dd221681b3fd16df65bf22cb89d1c6f8